candle-nn/src/activation.rs

```diff
@@ -29,7 +31,9 @@ impl super::Module for Activation {
             Self::Relu6 => xs.clamp(0f32, 6f32),
             Self::Silu => crate::ops::silu(xs),
             Self::Sigmoid => crate::ops::sigmoid(xs),
+            Self::HardSigmoid => crate::ops::hard_sigmoid(xs),
             Self::Swish => xs * crate::ops::sigmoid(xs)?,
+            Self::HardSwish => xs * crate::ops::hard_sigmoid(xs)?,
```

Comment:
Why does hard-swish point at hard-sigmoid? My understanding is that these should be two different functions.

Reply:
hard-switch = x * hard-sigmoid 
Details at: [https://pytorch.org/docs/stable/generated/torch.nn.functional.hardswish.html#torch.nn.functional.hardswish](url)

Reply:
Ah right, thanks for clarifying.